debian/raspberrypi-net-mods.service

```diff
@@ -1,13 +1,13 @@
 [Unit]
 Description=Copy user wpa_supplicant.conf
-ConditionPathExists=/boot/wpa_supplicant.conf
+ConditionPathExistsGlob=/boot/wpa_supplicant.conf{,.txt}
 Before=dhcpcd.service
 After=systemd-rfkill.service
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/bin/mv /boot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf
+ExecStart=ExecStart=/bin/sh -c "if [ -f "/boot/wpa_supplicant.conf" ]; then /bin/mv /boot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf; if [ -f "/boot/wpa_supplicant.conf.txt" ]; then /bin/rm /boot/wpa_supplicant.conf.txt; fi; elif [ -f "/boot/wpa_supplicant.conf.txt" ]; then /bin/mv /boot/wpa_supplicant.conf.txt /etc/wpa_supplicant/wpa_supplicant.conf; fi"
```

Comment:
I know nothing about systemd, but is the double `ExecStart=` here intentional?

Reply:
Well spotted - cut and paste error.

Reply:
I've not tested this modification myself, but do any of the double-quotes need to be escaped? Why are double-quotes-around-filenames used in some places but not others?